### PR TITLE
[Intel MKL] Upgrading MKL DNN to 0.18 RC

### PR DESCRIPTION
**Pull request by agramesh1**

### CODE REVIEW

File: tensorflow/workspace.bzl [hunk 1]
```diff
@@ -82,46 +82,48 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
...
+            "https://github.com/intel/mkl-dnn/releases/download/v0.18-rc/mklml_mac_2019.0.3.20190125.tgz",
         ],
     )
 
     if path_prefix:
         print("path_prefix was specified to tf_workspace but is no longer used " +
               "and will be removed in the future.")
 
+    # Important: If you are upgrading MKL DNN, then update the version numbers
+    # in third_party/mkl_dnn/mkldnn.BUILD.
```
Comment (penpornk) [inline tensorflow/workspace.bzl]: Could you please also add a note that the MKL ML files above also need to be updated if MKL-DNN is jumping a version?

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you very much, Ramesh! Could you please replace `MKL DNN` with `MKL-DNN`? I just want it to be consistent with the MKL-DNN repo.

**Comment by agramesh1** [comment]
@penpornk Thanks for the quick review. I modified the PR based on your comments.

**Review by penpornk — COMMENTED** [review]
Thank you for your quick changes, Ramesh! I noticed that MKL-DNN released v0.18 two hours ago. Would  you like to proceed with the current commit on the branch or change to the v0.18 commit? (I'm fine either way.)

**Review by penpornk — APPROVED** [review]
We talked offline and decided to go with this commit.

**Comment by penpornk** [comment]
@agramesh1 Merging is blocked because your workspace.bzl is now out of date. Could you please sync? Thank you!

**Comment by agramesh1** [comment]
@penpornk  I re-based the code.

**Review by penpornk — APPROVED** [review]
Thank you!

**Pull request merged**